### ISSUE TITLE
Some of the redesigned KDE apps. are not displaying properly

### ISSUE THREAD

**Issue by Trinity3e** [tsujan/Kvantum#879]
Hello,
I had the time to check if my custom themes were causing this, so I only set QT_QPA_PLATFORMTHEME, no style override, and in Kvantum the default theme. Stuff like spectacle still displays wrong.
It seems that for the redesign they broke some standard probably. As always excuse the potential sparsity of my question.

![img](https://github.com/tsujan/Kvantum/assets/123660226/cbdb7136-b780-4dca-9eab-908988b7c24a)

**Comment by tsujan** [comment]
Such apps are written in QML. You need to

 1. Run them only after setting the environment variable `QT_QUICK_CONTROLS_STYLE` to `org.kde.desktop`. That EV is automatically set under KDE. (Of course, you need `qqc2-desktop-style` outside KDE.)
 2. Apply the KDE color scheme that corresponds to the Kvantum theme you use. Each Kvantum theme should come with a matching KDE color scheme if it's supposed to be used with KDE apps, and especially with QML apps of KDE.

KDE devs are in love with QML and are ruining the GUIs of more and more apps by porting them to QML. At some point KDE might become QMLDE.

In short, this isn't related to Kvantum

**Comment by Trinity3e** [comment]
I get it, so QML/Kirigami is like a sort of libadawaita but for Qt. At least their submenus and stuff are untouched.
Thanks for the explanation, I did so and it now displays properly.
(also found out that KWin doesn't respect any env. var the desktop sets so I had to disable all of it's shortcuts/bindings as they conflicted with LXQt's)

**Comment by tsujan** [comment]
>  had to disable all of it's shortcuts/bindings as they conflicted with LXQt's

Oh, you use LXQt, like me.

For using KDE's QML apps inside LXQt, you could set `QT_QUICK_CONTROLS_STYLE` in LXQt Sessions Settings → Environment (Advanced). You also need to install enough KDE stuff to be able to apply KDE color schemes (lots of packages). And yes, you need to care for conflicting shortcuts (I've disabled all shortcuts of `kglobalaccel5` except for a few that are related to KWin).

**Comment by tsujan** [comment]
> I get it, so QML/Kirigami is like a sort of libadawaita but for Qt

Not exactly, but the overall trend has become like that of GNOME's.

**Comment by Trinity3e** [comment]
Here just systemsettings and yeah the qqc2-desktop-style package with it's deps, are needed to apply kde color theme from their menu (they call it application style) - kvantum. So yeah that was the missing link with this occasion. It's still somewhat of a continuity that they at least bothered to have with QML :-)

I think systemsettings can actually be removed once their weird convolution of config files gets set, this is what kde software looks for after all.

I set the env vars. and create user services all from LXQT's session menu, it's very solid and very simple, i started to love it a lot.

kglobalaccel5 and krunner gets pulled in with KWin as soon as it starts. I kinda wanted to use other WM's to avoid this but I lose window tilled snapping (or how else it's called :-) )

**Comment by tsujan** [comment]
> kglobalaccel5 and kRunner get started alongside KWin

Here only kglobalaccel5 gets started; krunner doesn't.

**Comment by Trinity3e** [comment]
I might've mistook it for the shortcut triggered by the keybinding, indeed, as a process (while in the background at least) it doesn't show up.

It's fine still, can't complain. My biggest wonder is why xfwm4 doesn't do snap tilling while outside of xfce desktop. Or of course, icewm can be scripted for that but it's way too buggy.
